doc/minutes/2022-12-01_first_meeting.md

```diff
@@ -0,0 +1,73 @@
+---
+title:  "First meeting YOS Award"
+date:   "30-11-2022"
+editor_options:
+  markdown:
+    canonical: false
+---
+
+**Attendees:** <names and surnames of attendees>
+
+-   Daniel Morillo Cuadrado
+
+-   Roberto Mediavilla Torres
+
+-   Cristina Rodríguez-Prada
+
+-   Lea Francia
+
+-   Jesus Sierralaya
+
+**Scribe:** Jesus Sierralaya
+
+**Schedule:** 11:00 - 11:30 <!-- # Init time - End time -->
+
+# Action points
+
+## Project Proposals
+
+### Robert's proposal
```

Comment:
These sections must go inside "Debate" (do not use new headings preferrably)
Also, it's not common practice to name the person that makes a proposal. We share accountability for the results as a whole team ;)